provider/terraform/tests/resource_dns_managed_zone_test.go

```diff
@@ -61,7 +61,7 @@ func TestAccDnsManagedZone_update(t *testing.T) {
 	})
 }
 
-func testAccCheckDnsManagedZoneDestroy(s *terraform.State) error {
+func testAccDnsManagedZoneDestroy(s *terraform.State) error {
```

Comment:
Had to rename this to deal with the name clash with the auto generated tests. Not sure if there's a better way.

Reply:
Actually, you can just get rid of this function from this file and let the tests here call that fn.